[antlr4,python3] Conflicting ports

Repro:

```
vcpkg.exe install antlr4
vcpkg.exe install python3 --editable
```

results in output like:

```
     3>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29333\include\cstdint(21,1): error C2061: syntax error: identifier 'std' [D:\buildtrees\python3\x86-windows-rel\v3.9.0-664bf2a259.clean\PCbuild\pythoncore.vcxproj]
     3>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29333\include\cstdint(21,1): error C2059: syntax error: ';' [D:\buildtrees\python3\x86-windows-rel\v3.9.0-664bf2a259.clean\PCbuild\pythoncore.vcxproj]
```

This is why the python3 port currently fails in our ci.

This is because Python wants to use a header named "token.h", and Antlr installs a Token.h. I got a preprocessed python file from both environments and diffed them to get this result; the broken build is on the left and the working one is on the right:

![image](https://user-images.githubusercontent.com/1544943/103313581-5a7dfd00-49d5-11eb-9a70-08cdd2c5342f.png)

--- a/ports/antlr4/CONTROL
+++ b/ports/antlr4/CONTROL
@@ -1,5 +1,6 @@
 Source: antlr4
 Version: 4.8
+Port-Version: 1
 Homepage: https://www.antlr.org
 Description: ANother Tool for Language Recognition
 Build-Depends: libuuid (!uwp&!windows&!osx)

--- a/ports/antlr4/portfile.cmake
+++ b/ports/antlr4/portfile.cmake
@@ -78,12 +78,6 @@ else()
     endif()
 endif()
 
-file(GLOB HDRS LIST_DIRECTORIES true ${CURRENT_PACKAGES_DIR}/include/antlr4-runtime/*)
-file(COPY ${HDRS} DESTINATION ${CURRENT_PACKAGES_DIR}/include)
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/include/antlr4-runtime)
-
 vcpkg_copy_pdbs()
 
 file(INSTALL ${LICENSE} DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
-
-message(STATUS "Installing done")

--- a/ports/python3/python_vcpkg.props.in
+++ b/ports/python3/python_vcpkg.props.in
@@ -6,7 +6,7 @@
       <PreprocessorDefinitions>_Py_HAVE_ZLIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="${VCPKG_LIBRARY_LINKAGE} == 'static'">XML_STATIC;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <AdditionalIncludeDirectories>
-        ${CURRENT_INSTALLED_DIR}/include;%(AdditionalIncludeDirectories)
+        %(AdditionalIncludeDirectories);${CURRENT_INSTALLED_DIR}/include
       </AdditionalIncludeDirectories>
 
       <RuntimeLibrary Condition="'${VCPKG_CRT_LINKAGE}|$(Configuration)' == 'static|Debug'">MultiThreadedDebug</RuntimeLibrary>

--- a/ports/python3/vcpkg.json
+++ b/ports/python3/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "python3",
   "version-string": "3.9.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "The Python programming language",
   "homepage": "https://github.com/python/cpython",
   "supports": "!(arm | uwp)",

